config/locales/en.yml

```diff
@@ -40,6 +40,8 @@ en:
     recommendation_step: "View the decision notice"
   full: "Full Householder Application"
   lawfulness_certificate: "Certificate of Lawfulness"
+  Proposed: "proposed"
+  Existing: "existing"
```

Comment:
This all looks great, with one minor thing: we store everything else like this as lowercase in the DB and API, and use i18n to `humanize` if needed when presenting it, and for this, we're doing the opposite. I think it would be best if we were consistent and did the same here?